eip/EIPS/eip-4337.md

```diff
@@ -91,7 +92,7 @@ The core interface required for a wallet to have is:
 interface IWallet {
   function validateUserOp
       (UserOperation calldata userOp, bytes32 requestId, address aggregator, uint256 missingWalletFunds)
-      external;
+      external returns (uint256 deadline);
```

Comment:
Should probably explain what this return value means.

And then also return this value during simulation so that the bundler can learn this value and drop expired ops without re-simulating them.  Maybe even mention that bundlers should defend themselves against expirations that are too short and unlikely to make it into a block.  And that they should keep track and drop expired ops without simulating again.

Also, before merging the "private" version of the EIP to the public one, be sure to merge with the public one.  There were some minor changes there since we last pushed from here.

Reply:
ok. didn't update the EIP comments yet, only the sol files..